Airports/EGNM/Stars.txt

```diff
@@ -1,4 +1,2 @@
 STAR:EGNM:32:COORD:POL
-STAR:EGNM:14:COORD:POL
-STAR:EGNM:32:COORD:
-STAR:EGNM:14:COORD:
```

Comment:
What effect will removing these lines have? Are they really redundant? (Genuinely don't know)

Reply:
From a rudimentary test it seemed to do nothing, those lines haven't been touched in 10 years since the `All airports (data unverified)` PR!

Reply:
@KHardern @hazzas-99 @cbyworth any of you know?

Reply:
Sounds good.